src/components/Card/Card.tsx

```diff
@@ -34,10 +34,14 @@ const Card = ({plant}: CardProps) => {
         })
     })
 
+    const draggedCardStyle = isDragging ? {opacity: '.4'} : {}
```

Comment:
love the reduced opacity, but we might need to shrink the dragged card to match the size of the cells?